src/drivers/actuators/voxl_esc/voxl_esc.hpp

```diff
@@ -133,6 +133,17 @@ class VoxlEsc : public ModuleBase<VoxlEsc>, public OutputModuleInterface
 
 	static constexpr uint16_t VOXL_ESC_NUM_INIT_RETRIES = 3;
 
+	static constexpr float    VOXL_ESC_GPIO_CTL_DISABLED_SETPOINT = -0.1f;
+	static constexpr float    VOXL_ESC_GPIO_CTL_THRESHOLD = 0.0f;
+
+	static constexpr uint32_t VOXL_ESC_GPIO_CTL = 0;
```

Comment:
Doesn't look like this one is ever used.

Reply:
removed